packages/expo/e2e/__tests__/whoami-test.ts

```diff
@@ -61,9 +61,9 @@ it('runs `npx expo whoami`', async () => {
   if (results.stderr) {
     expect(results.stderr.trim()).toBe('Not logged in');
   } else {
-    expect(results.stdout.trim()).toBe(expect.any(String));
+    expect(!!results.stdout.trim()).toBe(true);
```

Comment:
doesn't really matter but this works too and should print out the string in the event of a failure
```suggestion
    expect(results.stdout.trim()).toBeTruthy();
```